apps/edge/main.py

```
@@ -1,59 +1,64 @@
 from config import EDGE_PORT, host
-from database import edge_db, logs
+from database import edge_db, logs, init_db, close_db_connection
 from models import EdgeFunction
 import datetime
 from decorator import loggers_route  # type: ignore
 from runtime import create_build_function
 import pytz
 import random
 
-utc_now = datetime.datetime.now(pytz.UTC).strftime("%Y-%m-%d %H:%M:%S")
+utc_now = lambda: datetime.datetime.now(pytz.UTC).strftime("%Y-%m-%d %H:%M:%S")
```

Ruff (0.8.2)

Do not assign a `lambda` expression, use a `def`

Rewrite `utc_now` as a `def`

(E731)